### ISSUE TITLE
Gmail is having authentication problems. Some features may not work. Try  logging in  to fix the problem.

### ISSUE THREAD

**Issue by adithshenoy** [meetfranz/franz#1745]
Gmail is having authentication problems. Some features may not work. Try  logging in  to fix the problem.

### Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/6744743/72207461-16cb7400-34a2-11ea-88f0-d43946109a62.png)

### Possible Solution
I think it is related to cookies

### Your Environment
Mac OS Catalina

**Comment by adam-py** [comment]
Same error here, in the same environment.

Doesn't seem to limit functionality. Click on "Logging in" reloads gmail without the error, but it appears again after a while.

**Comment by adithshenoy** [comment]
Yes, there seems to be no problem with the functionality. I think the issue has got to do something with cookies but I'm not sure how to resolve it. Suggestions are welcome!

**Comment by Bentze** [comment]
That happens if you're blocking cross-site cookies.

**Comment by Shaurov05** [comment]
This happens due to an extension you are using. In my case, the extension was ModHeader which I use while working with rest API.
So, while using Gmail or other Google services, uncheck all 'Authorization token' in the ModHeader or other extension that you are using. See the attached picture to have a clear understanding.
![Screenshot from 2021-01-05 10-43-57](https://user-images.githubusercontent.com/56897591/103607282-b3ecab80-4f42-11eb-8e41-6bbf9e3937cd.png)

**Comment by Waleedov** [comment]
I have the same problem, what to do?
![image](https://user-images.githubusercontent.com/48418015/117455996-01f0da80-af48-11eb-91ac-21ac99565ef5.png)
